extensions/not-diamond/package.json

```
@@ -0,0 +1,124 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "not-diamond",
+  "title": "Not Diamond",
+  "description": "Not Diamond is an AI model router that automatically determines which LLM is best-suited to respond to any query, improving LLM output quality by combining multiple LLMs into a meta-model that learns when to call each LLM. This extension answers always with the best model depending on the prompt.",
+  "icon": "extension-icon.png",
+  "author": "dragos_nedelcu",
+  "categories": [
+    "Developer Tools",
+    "Productivity"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "not-diamond",
+      "title": "Chat Not Diamond",
+      "subtitle": "Route your prompts to the best model",
```

```suggestion
      "subtitle": "",
```
Should not be used to describe the command

---

hmm there seems to be a validation on the length for subtitle. This will throw an error on pipeline 
```
Error:   17:18  error  length must be >= 2, but got 0
Error:   17:18  error  does not match pattern '^[^\\s]+(?: [^\\s]+)*$'
```

any advice on what should I add on this one just to pass the checks? 🙏 thanks!

---

As advised, removed the line fixed this ✅

---

Yeah, sorry about that 🙂